src/FSharp.Stats/AlgTypes.fs

```diff
@@ -285,6 +285,23 @@ namespace FSharp.Stats
               | _ -> raise (new System.NotSupportedException("The element type '"+(typeof<'T>.ToString())+"' carried by this vector or matrix does not support the INormFloat<_> operation (i.e. does not have a registered numeric association that supports this type)"))
 
         let mkDenseMatrixGU ops arr = DenseMatrix(ops,arr)
+        let mkSparseMatrixGU ops (arr: 'a[,]) =
+            let m = arr |> Array2D.length1
```

Comment:
please change variable names to more descriptive ones

Reply:
Changed variable names